### ISSUE TITLE
NoClassDefFoundError for io.ebean.migration.runner.ScriptTransform with version 12.2.0

### ISSUE THREAD

**Issue by almothafar** [ebean-orm/ebean-dbmigration#82]
I simply got an issue with the latest version:

```
4) Error injecting constructor, java.lang.NoClassDefFoundError: io/ebean/migration/runner/ScriptTransform
```

![image](https://user-images.githubusercontent.com/13887564/99531268-27deee80-29ab-11eb-948c-3e3f7c9cfc57.png)

![image](https://user-images.githubusercontent.com/13887564/99531754-e3a01e00-29ab-11eb-9596-d328a5b4b0a7.png)

Not sure if I miss something but that happened in production runtime (distributed final classes and jars), and not happening in the development environment.

**Comment by rbygrave** [comment]
Ah sorry, 12.2.0 of ebean-migration is actually only compatible with the
next release of ebean (that I am looking to release today).

So yeah that is a poor choice of version number for this release of
ebean-migration.

What is going on here is we are sorting out the dependencies between
ebean-migration, ebean-ddl-generator (added a ebean-ddl-runner).

in doing that the ScriptTransform got moved into the separate
ebean-ddl-runner.

Thanks for spotting this.

Cheers, Rob.

**Comment by rbygrave** [comment]
I have released version 12.6.0 of Ebean.  So if you are keen please try ebean-migration 12.2.0 with ebean 12.6.0.

I can't remove ebean-migration 12.2.0 from maven central now (and re-release it as version 12.6.0) so I think we just have to live with this mistake and hope that people bump ebean to 12.6.0 or higher before using ebean-migration 12.2.0.

**Comment by almothafar** [comment]
@rbygrave thank you! 
The issue is gone with version ebean 12.6.0
